egs2/misp2021/asr1/local/run_wpe.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# _*_ coding: UTF-8 _*_
+import os
+import codecs
+import argparse
+import numpy as np
+from tqdm import tqdm
+import scipy.io.wavfile as wf
+from multiprocessing import Pool
+from nara_wpe.wpe import wpe_v8 as wpe
+from nara_wpe.utils import stft, istft
```

Comment:
Do we have to add additional requirements for nara-wpe as a package?

Reply:
Is the detection and download process for the nara_wpe package in local/enhancement.sh enough for handling this? 
`pip show -f nara_wpe >/dev/null || pip install nara_wpe`

Reply:
Looks good! thx